android/logto-client/src/main/kotlin/io/logto/sdk/android/type/LogtoConfig.kt

```diff
@@ -0,0 +1,13 @@
+package io.logto.sdk.android.type
+
+import io.logto.sdk.core.extension.ensureDefaultScopes
+
+class LogtoConfig(
+    val endpoint: String,
+    val clientId: String,
+    scope: List<String>? = null,
+    val resource: List<String>? = null,
+    val usingPersistStorage: Boolean = false,
+) {
+    val scope by lazy { scope.ensureDefaultScopes() }
```

Comment:
Wondering `ensureDefaultScopes` should it be a general string list method?  Is it callable by any string list variables? As it is only useful for scope list.

Reply:
this can be used for all `List<String>?` values.
But before using it, you should import `ensureDefaultScopes` first, so I think it's fine here.
Maybe I will be better to rename `StringListExt` to `ScopeListExt`.

Reply:
LOG-1420: Refactor: Rename `StringListExt` to `ScopeListExt`